### PR TITLE
Paragon StatusAlert deprecation 

### PR DESCRIPTION
**Pull request by abdullahwaheed**
### Ticket
[Migrate off deprecated Paragon components](https://github.com/openedx/frontend-wg/issues/102)

### What has changed
Removed deprecated `StatusAlert` component of paragon and updated it with `Alert`

### References
[Paragon StatusAlert](https://paragon-openedx.netlify.app/components/statusalert/)
[Paragon Alert](https://paragon-openedx.netlify.app/components/alert/)

### CODE REVIEW

**Review by ansabgillani — APPROVED** [review]

**Review by Ali-D-Akbar — APPROVED** [review]
Be sure to squash the commits.

**Comment by abdullahwaheed** [comment]
@Ali-D-Akbar cannot merge this, people with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) can merge this PR

**Pull request merged**
